pom.xml

```diff
@@ -43,10 +43,6 @@
 		<spring-cloud-context.version>2.0.0.M5</spring-cloud-context.version>
```

Comment:
Neither `spring-cloud-build` nor `spring-cloud-dependencies` specify versions for these `spring-cloud` dependencies.
I'd really like to point to some BOM to get rid of the versions here. Any other BOM I'm missing that contains these versions?

Reply:
Take a look here: https://github.com/spring-cloud/spring-cloud-release/blob/master/spring-cloud-dependencies/pom.xml#L18.

Seems for me they are there. What ma I missing ?

Reply:
The `spring-cloud-context` is a part of the `spring-cloud-commons-dependencies` BOM: https://github.com/spring-cloud/spring-cloud-commons/blob/6e9745b7a431ac2673ab5ee182f32a7e57310e87/spring-cloud-commons-dependencies/pom.xml#L42